aten/src/ATen/native/cudnn/ConvShared.cpp

```diff
@@ -565,6 +554,21 @@ Tensor cudnn_convolution_add_relu(
       /*device=*/kCUDA,
       /*pin_memory=*/c10::nullopt,
       /*memory_format=*/at::MemoryFormat::Contiguous);
+  return cudnn_convolution_relu_out(
+    input_t, weight_t, bias_t, stride, padding, dilation, groups, output_t);
+}
+
+Tensor cudnn_convolution_add_relu_out(
+    const Tensor& input_t,
+    const Tensor& weight_t,
+    const Tensor& z_t,
+    const c10::optional<Scalar>& alpha,
+    const c10::optional<Tensor>& bias_t,
+    IntArrayRef stride,
+    IntArrayRef padding,
+    IntArrayRef dilation,
+    int64_t groups,
+    Tensor& output) {
```

Comment:
I think you need to this parameter as `output_t`. You can use `python test/test_jit.py -k test_freeze_conv_relu_fusion` for testing because this set of ops are used in the conv-relu fusion. @eellison, is there any existing approach to add a unit test for this kind of new operators?

Reply:
yup (that's probably why lots of tests fail). this isn't in my compile path locally so I missed it but I was planning on double checking on devgpu before landing/shipping